packages/frontEnd/src/views/blogList/topCategories/TopCategories.tsx

```diff
@@ -0,0 +1,16 @@
+interface CategoriesPropCard {
+  key: string;
+  title: string;
+  value: string;
+}
+
+export const TopCategories = ({ key, title, value }: CategoriesPropCard) => {
+  return (
+    <div className="flex justify-between items-center border-b ">
+      <div className="text-gray-dark text-lg font-medium">{title}</div>
+      <div className="flex justify-center items-center text-dark-blue bg-white rounded-full my-2 font-medium w-[35px] h-[35px]">
+        <p className="text-sm ">{value}</p>
+      </div>
+    </div>
```

Comment:
It's missing hover.

Reply:
![image](https://user-images.githubusercontent.com/60447475/197564925-7c23b083-2408-4283-9037-0f7cc05a31d5.png)